src/chains/filecoin/filecoin/webpack/webpack.common.config.ts

```diff
@@ -0,0 +1,66 @@
+import webpack from "webpack";
+import TerserPlugin from "terser-webpack-plugin";
+
+const base: webpack.Configuration = {
+  mode: "production",
+  entry: "./index-webpack.ts",
+  devtool: "source-map",
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: [
+          {
+            loader: "ts-loader",
+            options: {
+              // we need to use ttypescript because we use ts transformers
+              compiler: "ttypescript",
+              // Symlinked paths to our packages aren't resolving correctly...
+              // E.g., if PackageA and PackageB both import PackageC, the
+              // compiler assumes PackageA's PackageC is incompatible with
+              // PackageB's PackageC.
+              // Note: if all packages are precompiled before running webpack
+              // this issue doesn't occur, which makes me think this might be a
+              // ts-loader issue.
+              transpileOnly: true
+            }
+          }
+        ]
+      },
+      {
+        test: /\.jsx?$/,
+        use: [
+          {
+            loader: "babel-loader"
+          }
+        ]
+      }
```

Comment:
I added this at one point to babelify nodejs dependencies; we don't have anything internal that needs this. Some dependencies since then were set to be external, so it's possible that I may be able to get rid of this. I could give it a shot